engine/storage/src/main/java/org/apache/cloudstack/storage/allocator/ClusterScopeStoragePoolAllocator.java

```diff
@@ -107,7 +107,7 @@ protected List<StoragePool> select(DiskProfile dskCh, VirtualMachineProfile vmPr
             }
         }
 
-        s_logger.debug(String.format("ClusterScopeStoragePoolAllocator is returning [%s] suitable storage pools [%s].", suitablePools.size(), suitablePools));
+        logEndOfSearch(suitablePools);
```

Comment:
code lgtm

however, I like the old style more than the new method `logEndOfSearch`

Reply:
I know it. I just think it is unnecessary. it will not bring any benefit I think.
on the contrary, if I get log "ClusterScopeStoragePoolAllocator is returning xxx" and grep in source code, the result will be empty.
ps: this does not mean -1 to this pr.